### PR TITLE
Move all SWF tests out of `core` to new `tests` package

### PR DESCRIPTION
**Pull request by Aaron1011**
This allows `regressions_tests.rs` to depend on other crates in the
workspace, such as `render`, without introducing a cyclic dependency.

Split out from #4054

### CODE REVIEW

**Comment by Herschel** [comment]
Looks like you accidentally caught a few of the rendering reference files in here `tests/color-linux-SwiftShader Device (LLVM 10.0.0).png` etc.)

File: tests/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+version = "0.1.0"
+authors = ["Aaron Hill <aa1ronham@gmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+ruffle_core = { path = "../core" }
+ruffle_render_wgpu = { path = "../render/wgpu" }
+wgpu = { git = "https://github.com/gfx-rs/wgpu-rs", rev = "50e0577" }
```
Comment (Herschel) [inline tests/Cargo.toml:12]: This should be removed and kept in #4054

**Comment by Aaron1011** [comment]
@Herschel: I've addressed your comments

**Review by Herschel — APPROVED** [review]

**Comment by Herschel** [comment]
Thank you!

**Pull request merged**